### PR TITLE
Update documentation on page variables

### PR DESCRIPTION
**Pull request by jamezrin**

### CODE REVIEW

File: content/en/variables/page.md [hunk 1]
```diff
@@ -168,6 +168,12 @@ http://remarkjs.com)
 .Translations
 : a list of translated versions of the current page. See [Multilingual Mode](/content-management/multilingual/) for more information.
 
+.TranslationKey
+: the key used to map language translations of the current page. See [Multilingual Mode](/content-management/multilingual/) for more information.
+
+.LinkTitle
```
Comment (bep) [inline content/en/variables/page.md]: LinkTitle is already up there.

Comment (jamezrin) [reply content/en/variables/page.md]: Sorry, corrected.

File: content/en/variables/page.md [hunk 2]
```diff
@@ -60,6 +63,9 @@ See [`.Scratch`](/functions/scratch/) for page-scoped, writable variables.
 .Hugo
 : see [Hugo Variables](/variables/hugo/).
```
Comment (bep) [inline content/en/variables/page.md]: Images and Videos are undocumented for a reason. The design wasn't really thought through and we will replace it with "remote resources" or some thing, so we get "one thing".

Comment (jamezrin) [reply content/en/variables/page.md]: Understandable, I dropped these commits. I'm going to go over and check what remaining variables I can add that have documentation.

**Comment by jamezrin** [comment]
Please let me know if I should add more variables and I will add them

**Comment by jamezrin** [comment]
@bep Any comments?

File: content/en/variables/page.md [hunk 3]
```diff
@@ -30,6 +30,15 @@ See [`.Scratch`](/functions/scratch/) for page-scoped, writable variables.
 .AlternativeOutputFormats
 : contains all alternative formats for a given page; this variable is especially useful `link rel` list in your site's `<head>`. (See [Output Formats](/templates/output-formats/).)
 
+.Aliases
+: aliases of this page
+
+.Author
```
Comment (bep) [inline content/en/variables/page.md]: Drop Author, Authors and IsSection from this list (for different reasons)

Comment (jamezrin) [reply content/en/variables/page.md]: OK, dropped these commits.

**Comment by jamezrin** [comment]
@bep Anything else I should do?

**Comment by jamezrin** [comment]
There is still no documentation about these variables

**Pull request merged**